lib/assessment.rb

```diff
@@ -6,13 +6,23 @@
 # Make all of those attributes read-only.
 
 ##
-# your code here
+class Dog
+  attr_reader :name
+  attr_reader :breed
+  attr_reader :age
+
+  def initialize name, breed, age
+    @name = name
+    @breed = breed
+    @age = age
+  end
+end
```

Comment:
Yep, this is right. By the way, did you know that you can write all of those `attr_reader`s on a single line?
e.g.
`attr_reader :name, :breed, :age`